$DragonFly: src/gnu/usr.bin/cvs/lib/Attic/xgethostname.c.patch,v 1.1 2005/04/18 20:22:42 dillon Exp $
--- ../../../../contrib/cvs-1.12.11/lib/xgethostname.c	2004-12-18 13:54:14.000000000 -0800
+++ xgethostname.c	2005-04-18 11:30:43.000000000 -0700
@@ -69,7 +69,7 @@
 	  if (! hostname[size_1 - 1])
 	    break;
 	}
-      else if (errno != 0 && errno != ENAMETOOLONG && errno != EINVAL)
+      else if (errno != 0 && errno != ENAMETOOLONG && errno != EINVAL && errno != ENOMEM)
 	{
 	  int saved_errno = errno;
 	  free (hostname);
